api/kubernetes/helm-chart/values.yaml

```diff
@@ -162,6 +162,9 @@ argo:
     password: ""
 
 config:
+  secrets:
+    from_env:
```

Comment:
Could you please rename `from_env` in `envFrom` please?

Reply:
done